tests/test_full_text_search.py

```diff
@@ -192,6 +221,27 @@ def test_search_by_name_only(self):
         assert set([r.record_id for r in results]) == \
             set(['haiti/0225'])
 
+        # Search romaji record by hiragana name and location
+        results = full_text_search.search('haiti', u'ちはや あらお', 5)
+        assert set([r.record_id for r in results]) == \
+            set(['haiti/0225'])
+
+        # Search by full name without space
+        results = full_text_search.search('haiti', 'HibikiGanaha', 5)
+        assert set([r.record_id for r in results]) == \
+            set(['haiti/1010'])
+
+        # Search by full name without space
```

Comment:
Can you update comment here and line 239, to clarify how these are different from the previous test?